script.js

```diff
@@ -42,33 +42,41 @@ function formatQuery(params) {
 
 function displayRecipes(responseJson) {
     console.log(responseJson)
-    for (let i = 0; i < responseJson.hits.length; i++) {
-        let ingredients = responseJson.hits[i].recipe.ingredientLines.map((a) => {
-            return `<li>${a}</li>`
-        });
-        $('#js-results').append(`<div class="recipe"><h3>${responseJson.hits[i].recipe.label}</h3>
-        <img src="${responseJson.hits[i].recipe.image}" class="child"> 
-          <ul>
-          <li>Serving Size:</li>
-          <li>Diet Labels:${responseJson.hits[i].recipe.healthLabels}</li>
-          </ul>
-        <hr>
-        <h4>Ingredients</h4>
-        <ul id="js-ingredients">
-        ${ingredients.join('')}
-        </ul>
-        <a href="${responseJson.hits[i].recipe.url}" class="button" target="_blank">View Instructions</a>
-        <button id="js-beer-results">Click for Beer Recs</button> 
-            <div class="js-beer-results-list hidden"></div>
-        </div>`
-        )
-    };
+    // we can't always assume responseJson will have the .hits property
+    let { hits = [] } = responseJson;
+    if (hits.length > 0) {
+        for (let i = 0; i < hits.length; i++) {
+            let ingredients = responseJson.hits[i].recipe.ingredientLines.map((a) => {
+                return `<li>${a}</li>`
+            });
+            // TODO: add alt attribute to img tag below
+            $('#js-results').append(`<div class="recipe"><h3>${responseJson.hits[i].recipe.label}</h3>
+                <img src="${responseJson.hits[i].recipe.image}" class="child" alt="temp"> 
+                <ul>
+                <li>Serving Size:</li>
+                <li>Diet Labels:${responseJson.hits[i].recipe.healthLabels}</li>
+                </ul>
+                <hr>
+                <h4>Ingredients</h4>
+                <ul id="js-ingredients-${i}">
+                    ${ingredients.join('')}
+                </ul>
+                <a href="${responseJson.hits[i].recipe.url}" class="button" target="_blank">View Instructions</a>
+                <button id="js-beer-results-${i}" class="js-beer-results">Click for Beer Recs</button> 
```

Comment:
notice here, that I've added the index `i` onto the button id. This is because it's invalid HTML to have multiple HTML elements with the same id.